source/val/validation_state.cpp

```diff
@@ -425,4 +425,7 @@ void ValidationState_t::RegisterSampledImageConsumer(uint32_t sampled_image_id,
   sampled_image_consumers_[sampled_image_id].push_back(consumer_id);
 }
 
+uint32_t ValidationState_t::getIdBound() const { return id_bound_; }
```

Comment:
It's good to match surrounding code, but you should know that Google C++ style guide permits (encourages?) accessors like this to just use the root of the member name.
E.g . in this case it would be    id_bound().  
Also, it's perfectly fine to put the body in the class declaration, and hence be available for inlining.  Similar for the setter.
See last paragraph in https://google.github.io/styleguide/cppguide.html#Function_Names